src/PrestaShopBundle/Controller/Admin/Improve/Modules/AddonsStoreController.php

```diff
@@ -67,14 +77,17 @@ public function indexAction(Request $request)
      */
     private function getAddonsUrl(Request $request)
     {
-        $psVersion = $this->get('prestashop.core.foundation.version')->getVersion();
-        $parent_domain = $request->getSchemeAndHttpHost();
         $context = $this->getContext();
-        $currencyCode = $context->currency->iso_code;
-        $languageCode = $context->language->iso_code;
-        $countryCode = $context->country->iso_code;
-        $activity = (int) $this->get('prestashop.adapter.legacy.configuration')->get('PS_SHOP_ACTIVITY');
 
-        return "https://addons.prestashop.com/iframe/search-1.7.php?psVersion=$psVersion&isoLang=$languageCode&isoCurrency=$currencyCode&isoCountry=$countryCode&activity=$activity&parentUrl=$parent_domain";
+        $queryParameters = [
```

Comment:
I remember to have done it 2 years ago, and it ended wrong... don't remember why though :thinking:

Reply:
I have used `http_build_query`. This will encode parameters for an URL, which was not performed by the original processing.

So the code before would output
`https://addons.prestashop.com/iframe/search-1.7.php?psVersion=176&isoLang=fr&isoCurrency=EUR&isoCountry=FR&activity=foo&parentUrl=http://www.google.com`
while my PR would output
`https://addons.prestashop.com/iframe/search-1.7.php?psVersion=176&isoLang=fr&isoCurrency=EUR&isoCountry=FR&activity=foo&parentUrl=http%3A%2F%2Fwww.google.com`

Is that an issue ?

Reply:
I remember now! The Addons server wasn't able to handle the well formatted uris, can you check with Denis?

Reply:
Woot, I remeber this PR: https://github.com/PrestaShop/PrestaShop/pull/9255/files#diff-4f0928deabb59e9c6fd5e5c675628d61L51

Reply:
:cry: